operator-pipeline-images/operatorcert/pyxis.py

```diff
@@ -386,3 +386,52 @@ def wait_for_image_request(
             )
             return image_request
         time.sleep(delay)
+
+
+def wait_for_container_grades(
+    base_url: str, image_identifier: str, timeout: float = 60 * 10, delay: float = 5
+) -> Any:
```

Comment:
Shouldn't this simply return None? I can see it returns `container_image`, is there reason for that except for testing purpose?

Reply:
The return value here is for consistency reasons as there are other functions in the module that have the same logic and return values.